diff -rupN audaspace-1.8.0.old/src/sequence/AnimateableProperty.cpp audaspace-1.8.0/src/sequence/AnimateableProperty.cpp
--- audaspace-1.8.0.old/src/sequence/AnimateableProperty.cpp	2025-11-13 08:09:33.000000000 +0100
+++ audaspace-1.8.0/src/sequence/AnimateableProperty.cpp	2026-03-15 21:02:52.371542322 +0100
@@ -223,8 +223,7 @@ void AnimateableProperty::read(float pos
 			m0 = (p2[i] - p0[i]) / 2.0f;
 			m1 = (p3[i] - p1[i]) / 2.0f;
 
-			out[i] = (2 * t3 - 3 * t2 + 1) * p0[i] + (-2 * t3 + 3 * t2) * p1[i] +
-					 (t3 - 2 * t2 + t) * m0 + (t3 - t2) * m1;
+			out[i] = (2 * t3 - 3 * t2 + 1) * p1[i] + (-2 * t3 + 3 * t2) * p2[i] + (t3 - 2 * t2 + t) * m0 + (t3 - t2) * m1;
 		}
 	}
 }
