diff --git a/bindings/python/setup.py.in b/bindings/python/setup.py.in
index 949e12b..357c8e3 100644
--- a/bindings/python/setup.py.in
+++ b/bindings/python/setup.py.in
@@ -39,7 +39,7 @@ else:
 audaspace = Extension(
                       'aud',
                       include_dirs = ['@CMAKE_CURRENT_BINARY_DIR@', '@FFTW_INCLUDE_DIR@', os.path.join(source_directory, '../../include'), numpy.get_include()],
-                      libraries = ['audaspace'],
+                      libraries = ['audaspace', 'python%s.%s' % (str(sys.version_info.major), str(sys.version_info.minor))],
                       library_dirs = ['.', 'Release', 'Debug'],
                       language = 'c++',
                       extra_compile_args = extra_args,
